src/main/java/frc/team4909/robot/sensors/Stream.java

```diff
@@ -1,103 +1,76 @@
-      camera2.setFPS(12);
-      CvSink cvSink1 = CameraServer.getInstance().getVideo();
-      CvSource outputStream = CameraServer.getInstance().putVideo("Source 1", 160, 120);
-      CvSink cvSink2 = CameraServer.getInstance().getVideo();
-      CvSource outputStream2 = CameraServer.getInstance().putVideo("Source 2", 160, 120);
-      camera3.setResolution(120, 90);
-      camera4.setResolution(160, 120);
-      camera3.setFPS(12);
-      camera4.setFPS(12);
-      CvSink cvSink3 = CameraServer.getInstance().getVideo();
-      CvSource outputStream3 = CameraServer.getInstance().putVideo("Source 3", 160, 120);
-      CvSink cvSink4 = CameraServer.getInstance().getVideo();
-      CvSource outputStream4 = CameraServer.getInstance().putVideo("Source 4", 160, 120);
-      //Implement toggle between two cameras between two streams with Button X
-      Mat source1 = new Mat();
-      Mat source2 = new Mat();
-      Mat output1 = new Mat();
-      Mat output2 = new Mat();
-      Mat source3 = new Mat();
-      Mat source4 = new Mat();
-      Mat output3 = new Mat();
-      Mat output4 = new Mat();
-      /*
-      Mat image1 = new Mat(cvSink1.grabFrame(source1));
-      int height1 = image1.rows();
-      int length1 = image1.cols();
-      MatOfPoint2f src = new MatOfPoint2f(
-                new Point((1/6)*height1, 0),
-                new Point((5/6)*height1,0),
-                new Point(height1-1,length1-1),
-                new Point(0,length1-1)
-                );
-
-      MatOfPoint2f dst = new MatOfPoint2f(
-                new Point(0, 0),
-                new Point(height1-1,0),
-                new Point(0,length1-1),
-                new Point(height1-1,length1-1)      
-                );
-      Mat hatchWatchMatrix = Imgproc.getPerspectiveTransform(src, dst);
+      //Cameras 3 and 4 have mutual toggle function.
+      UsbCamera camera0 = CameraServer.getInstance().startAutomaticCapture(0); //Function: Drive
+      UsbCamera camera1 = CameraServer.getInstance().startAutomaticCapture(1); //Function: Line
+      UsbCamera camera2 = CameraServer.getInstance().startAutomaticCapture(2); //Function: Hatch 
+      UsbCamera camera3 = CameraServer.getInstance().startAutomaticCapture(3); //Function: Cargo
+      camera0.setResolution(160, 120);    /******************************************** */
+      camera1.setResolution(160, 120);    /**********************  Set   ************** */
+      camera0.setFPS(21);                 /********************  Camera  ************** */  
+      camera1.setFPS(16);                 /*************  Specifications    *********** */
+      camera2.setResolution(160, 120);    /******************************************** */
+      camera3.setResolution(160, 120);    /******************************************** */
+      camera2.setFPS(10);                 /******************************************** */
+      camera3.setFPS(10);                 /******************************************** */
```

Comment:
For organization, it's much easier to see how multiple things are configured if you group them appropriately. I can see two reasonable groups here: do all of the camera0 things, then all of the camera1 things, so that would yield something like this:
```
      // Instantiate and configure camera 0: Drive
      UsbCamera camera0 = CameraServer.getInstance().startAutomaticCapture(0);
      camera0.setResolution(160, 120);
      camera0.setFPS(21);

      // Instantiate and configure camera 1: Line
      UsbCamera camera1 = CameraServer.getInstance().startAutomaticCapture(1);
      camera1.setResolution(160, 120);
      camera1.setFPS(16);

      // Instantiate and configure camera 2: Hatch
      UsbCamera camera2 = CameraServer.getInstance().startAutomaticCapture(2);
      camera2.setResolution(160, 120);
      camera2.setFPS(10);

      // Instantiate and configure camera 3: Cargo
      UsbCamera camera3 = CameraServer.getInstance().startAutomaticCapture(3);
      camera3.setResolution(160, 120);
      camera3.setFPS(10);
```
or group them by function:
```
      // Instantiate the cameras
      UsbCamera camera0 = CameraServer.getInstance().startAutomaticCapture(0); //Function: Drive
      UsbCamera camera1 = CameraServer.getInstance().startAutomaticCapture(1); //Function: Line
      UsbCamera camera2 = CameraServer.getInstance().startAutomaticCapture(2); //Function: Hatch 
      UsbCamera camera3 = CameraServer.getInstance().startAutomaticCapture(3); //Function: Cargo

      // Set resolutions for each camera
      camera0.setResolution(160, 120);
      camera1.setResolution(160, 120);
      camera2.setResolution(160, 120);
      camera3.setResolution(160, 120);

      // Set frame rates for each camera
      camera0.setFPS(21);
      camera1.setFPS(16);
      camera2.setFPS(10);
      camera3.setFPS(10);
```
Please select one of these organizations, so that it's easy to see at quick glance how things are configured.